tools/gulp/util/package-build.ts

```diff
@@ -0,0 +1,111 @@
+import {join, basename, dirname, relative} from 'path';
+import {main as tsc} from '@angular/tsc-wrapped';
+import {DIST_BUNDLES, DIST_ROOT, SOURCE_ROOT, PROJECT_ROOT, LICENSE_BANNER} from '../constants';
+import {createRollupBundle} from '../util/rollup-helper';
+import {inlineMetadataResources} from '../util/inline-resources';
+import {transpileFile} from '../util/ts-compiler';
+import {ScriptTarget, ModuleKind} from 'typescript';
+import {sync as glob} from 'glob';
+import {
+  writeFileSync, copySync, mkdirpSync, readFileSync
+} from 'fs-extra';
+
+// There are no type definitions available for these imports.
+const uglify = require('uglify-js');
+
+export function composeRelease(packageName: string) {
```

Comment:
Add function description? It could be good to document each of thing things that get packaged into a release.